ash/src/extensions/ext/debug_utils.rs

```diff
@@ -20,7 +20,7 @@ impl DebugUtils {
     }
 
     /// <https://www.khronos.org/registry/vulkan/specs/1.3-extensions/man/html/vkSetDebugUtilsObjectNameEXT.html>
-    #[deprecated = "Backwards-compatible alias containing a typo, use [`set_debug_utils_object_name`] instead"]
+    #[deprecated = "Backwards-compatible alias containing a typo, use `Self::set_debug_utils_object_name()` instead"]
```

Comment:
Sorry, unclear on my part: drop the `Self::` bit, that's only relevant if this were an intradoc link (which, as we clarified before, doesn't work).

Reply:
Ok, the thing is I don't know how to check if it works or not since the IDE I'm using does not support it :)

Reply:
Nothing supports this, apparently; both `rustdoc` and `rust-analyzer` don't show an intradoc links for this (per screenshot above).

To explain again: backticks render the text as monospace. For intradoc links - when adding `[]` around it - `rustdoc` looks in the current scope for the symbol, but these associate methods are only available under `DebugUtils::` (and because we're "in" `DebugUtils`, `Self` works as an alias to that). However, since intradoc links are not resolved in `deprecated = "..."` messages there shouldn't be `[]` _nor_ `Self::` / `DebugUtils::`, that only looks funky to the user.